fmatch/matcher.py

```diff
@@ -95,7 +97,7 @@ def get_uuid_by_metadata(
         if index is None:
             index = self.index
 
-        version = meta["ocpVersion"][:4]
+        version = str(meta[version_field])[:4]
```

Comment:
This logic might not apply to every other product version. We need some regex to get version numbers out.

Reply:
This getting of the 4 characters of the version was already there. Are we able to change that in a future PR when a product version needs it?